ultradns/client.py

```diff
@@ -16,13 +17,15 @@
 ERR_RECORDS_NOT_FOUND = 70002
 ERR_RECORD_ALREADY_EXISTS = 2111
 ERR_RECORD_NOT_FOUND = 56001
+ERR_HTTP = 111222333
```

Comment:
We need some error code for such kind of errors. DNS-level errors have their codes assigned by UltraDNS. This one is new.